Dockerfile.rh

```diff
@@ -0,0 +1,8 @@
+FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:v1.23.4@sha256:5c6607f82aeb2cd3f71075d17555d4170b17927e8551967d2fd0e534b22b1c7b
+
+WORKDIR /
+
+COPY . .
+
+RUN go build -o /rekor_monitor ./cmd/rekor_monitor
+ENTRYPOINT ["/rekor_monitor"]
```

Comment:
Modify it to multistage build dockerfile and for final stage use some UBI9 image